docker-compose.yml

```diff
@@ -10,13 +10,21 @@ services:
       - PORT=8080
     ports:
       - 8080:8080
-  
+    volumes:
+      - ./api:/root/app
+
```

Comment:
Here we're telling `docker-compose` to mount the `./api` directory as a volume _within_ the container. This will (eventually) allow us to do hot-reloading. You change a file, it rebuilds and reloads the app within the container.